style.css

```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;700&display=swap');
```

Comment:
- [ ] Please I highly recommend sticking to the [Figma Design Template] per project requirements. 😄 

_**REQUIRED FIGMA DESIGN TEMPLATE**_

![Screenshot (96)](https://user-images.githubusercontent.com/78104544/227555927-6fedcc73-92ed-4d2c-8c7a-0f31abc900e8.jpg)


_**YOUR DESIGN**_

![Screenshot (94)](https://user-images.githubusercontent.com/78104544/227556088-e8386de1-75f7-4a19-9fb2-01e6c0f7b9a2.jpg)

Reply:
@braincee , good to see your review, i did that but as per the previous reviewer's comment, I have removed that portion. Now, you are asking to do it again, i do not know why?

Reply:
@braincee , for your information
<img width="608" alt="image" src="https://user-images.githubusercontent.com/118947090/227561658-2a6b1976-440c-49b0-b430-f0d7ad9d9e92.png">

Reply:
Please your design is to fix the mobile version screen. In addition to the design, you are supposed to add `MY LOGO` at the top left corner of the navigation bar